crates/rattler_installs_packages/src/index/package_database.rs

```diff
@@ -263,15 +269,45 @@ impl PackageDb {
                     .get_cached_artifact::<SDist>(artifact_info, CacheMode::Default)
                     .await?;
 
-                return builder.build_wheel(&sdist).await.into_diagnostic();
+                let whl = builder
+                    .build_wheel(&sdist)
+                    .await
+                    .into_diagnostic()
+                    .expect("cannot build wheel");
```

Comment:
Should probably not be an `expect`, right?

Reply:
yes, should be ?
thanks for spotting it!